tests/table/tableView.test.ts

```diff
@@ -423,5 +423,78 @@ describe("Table view", () => {
 
             expect(tableView.tablePlaces.place1().BackCards()).toEqual(allBacksClassesFourCards);
         });
+
+    });
+    describe("Hightlight omaha cards every time", () => {
+        beforeAll(() => {
+            GameActionsQueue.waitDisabled = true;
+            GameActionsQueue.drainQueuePause = 0;
+            debugSettings.tableView.trace = false;
+        });
+        afterAll(() => {
+            GameActionsQueue.waitDisabled = false;
+            GameActionsQueue.drainQueuePause = 100;
+        });
+        it("Cards Hightlighted should be true", async () => {
+            const winner: GameWinnerModel[] = [
+                {
+                    Amount: 100,
+                    Pot: 1,
+                    CardsDescription: "",
+                    PlayerId: 101
+                }
+            ]
+            const tableView = new TableView(1, {
+                TableId: 1,
+                TableName: "",
+                BigBlind: 200,
+                SmallBlind: 100,
+                CurrencyId: 1,
+                HandsPerHour: 0,
+                AveragePotSize: 0,
+                JoinedPlayers: 2,
+                MaxPlayers: 8,
+                PotLimitType: 2,
+            });
+            const players: GamePlayerStartInformation[] = [{
+                PlayerId: 1,
+                Money: 10000,
+            }, {
+                PlayerId: 2,
+                Money: 10000,
+            }];
+            const actions: GameActionStartInformation[] = [];
+            login("player1");
+            loginId(1);
+            const tableSatusPlayers = [
+                getSeatPlayer(1, 10000),
+                getSeatPlayer(2, 10000),
+            ];
+            tableView.onTableStatusInfo(tableSatusPlayers, [], null, 4, 100, 10, null, null, null, null, 1, true, 0, false, true, null, 0, 2);
+            tableView.onGameStarted(1, players, actions, 4);
+            tableView.onBet(2, 0, 100, 1);
+            tableView.onBet(1, 0, 200, 2);
+            tableView.onPlayerCards(1, [1, 2]);
+            tableView.onPlayerCards(2, [2, 3]);
+            tableView.onBet(2, 2, 50, 1);
+            tableView.onBet(1, 2, 50, 1);
+            //Flop
```

Comment:
You should move money to pot and display cards using `executeMoveToPot` and `onOpenCards`. See other tests for example.